src/pages/coaches/paul-woodley.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Paul Woodley
+title: Head of UX / Product Design
+companyName: LeanJS
+companyLink: https://www.leanjs.com/
+gitHub: https://github.com/shakersinthedark
+medium: https://medium.com/@paulw_design
+twitter: https://twitter.com/paulw_design
+linkedIn: https://uk.linkedin.com/in/paulwoodleydesign
+blockquote: I'm not a developer, but code is what is used to bring design to life so it's important everyone involved understands each other!
+imageDescription: React GraphQL Academy coach Francisco Gomes, smiling with sunglasses on
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactgraphqlacademy.appspot.com/o/images%2Fabout_us_profilePics%2FPaul_Profile.jpg?alt=media
+tags: []
+---
+
+ <ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists